### ISSUE TITLE
GitHub icon in the footer is not a link

### ISSUE THREAD

**Issue by encukou** [pradyunsg/lutra#37]
On both https://pradyunsg.me/lutra/ and https://pradyunsg-cpython-lutra-testing.readthedocs.io/en/latest/index.html, the GitHub icon at the bottom looks like it should be a link, but isn't.

**Comment by pradyunsg** [comment]
Yup, the footer icons are still not implemented -- the icon is currently hard-coded to a GitHub icon; mostly as a reminder to myself to come back and address that missing piece.
